scripts/dashboard/install-cephadm-e2e-deps.sh

```diff
@@ -76,7 +76,7 @@ echo "#!/usr/bin/env bash
 
 docker run --net host --security-opt label=disable \
     -v ${KCLI_CONFIG_DIR}:/root/.kcli \
-    -v ${PWD}:/workdir \
+    -v ${PWD}/ceph:/workdir \
```

Comment:
@nizamial09 This script should also run in a local environment, and AFAIK in a local environment this extra folder does not exist.

Reply:
Updated, thanks.